scripts/check-command-effects.py

```
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""Reject host effects outside the command effect interpreter."""
+
+from pathlib import Path
+from re import Pattern, compile
```

Ruff (0.15.21)

[error] Import `compile` is shadowing a Python builtin

(A004)